components/datepicker/src/auro-datepicker.js

```diff
@@ -374,30 +374,6 @@ export class AuroDatePicker extends LitElement {
     return dateStr;
   }
 
-  /**
-   * Return appropriate error message.
-   * @param {Object} evt - Event passed in from auro-input when the event triggered this function.
-   * @private
-   */
-  getErrorMessage(evt) {
-    if (evt) {
-      const inputClass = evt.target.getAttribute('class');
-      if (inputClass === 'dateFrom') {
-        if (this.inputList[0].validity && this.inputList[0].validity !== 'valid') {
-          this.errorMessage = evt.target.errorMessage;
-        } else {
-          this.errorMessage = undefined;
-        }
-      }
-
-      if (inputClass === 'dateTo') {
-        if (!this.errorMessage && this.inputList[1].validity && this.inputList[1].validity !== 'valid') {
-          this.errorMessage = evt.target.errorMessage;
-        }
-      }
-    }
-  }
-
```

Comment:
Isn't this still necessary to determine which input to pull the validation message from in range support?

Reply:
No because datepicker listens to input's `auroFormElement-validated` event and updates it's error message there. This function was not being used anywhere in the repo.

Reply:
Makes sense after reviewing.